src/Sensors.cpp

```diff
@@ -1546,6 +1571,30 @@ void Sensors::GCJA5Init() {
     sensorRegister(SENSORS::SGCJA5);
 }
 
+
+
+void Sensors::DFRobotgravityInit() {
+    sensorAnnounce(SENSORS::SMULTIGAS);
+     DFRobot_GAS_I2C gas(&Wire ,0x77);
```

Comment:
I think that is wrong. I'll review that, because the gas variable here is local variable. Like other inits we should instance the global object here. The dft_gas object. @roberbike